jaeger-mixin/alerts.libsonnet

```diff
@@ -14,27 +14,27 @@ local percentErrsWithTotal(metric_errs, metric_total) = '100 * sum(rate(%(metric
       {
         name: 'jaeger_alerts',
         rules: [{
-          alert: 'JaegerHTTPServerErrs',
-          expr: percentErrsWithTotal('jaeger_agent_http_server_errors_total', 'jaeger_agent_http_server_total') + '> 1',
+          alert: 'JaegerAgentUDPPacketsBeingDropped',
+          expr: 'rate(jaeger_agent_thrift_udp_server_packets_dropped_total[1m]) > 1',
           'for': '15m',
           labels: {
             severity: 'warning',
           },
           annotations: {
             message: |||
-              {{ $labels.job }} {{ $labels.instance }} is experiencing {{ printf "%.2f" $value }}% HTTP errors.
+              {{ $labels.job }} {{ $labels.instance }} is dropping {{ printf "%.2f" $value }} UDP packets per second.
             |||,
           },
         }, {
-          alert: 'JaegerRPCRequestsErrors',
-          expr: percentErrs('jaeger_client_jaeger_rpc_http_requests', 'status_code=~"4xx|5xx"') + '> 1',
+          alert: 'JaegerAgentHTTPServerErrs',
+          expr: percentErrsWithTotal('jaeger_agent_http_server_errors_total', 'jaeger_agent_http_server_total') + '> 1',
```

Comment:
IMO its borderline whether the benefits of functions like this are worth it.  Makes queries more consistent vs not obvious what the actual query is.